stacker/actions/build.py

```diff
@@ -271,26 +271,10 @@ def _handle_missing_parameters(self, params, required_params,
         return params.items()
 
     def _generate_plan(self, tail=False):
-        plan_kwargs = {}
-        if tail:
-            plan_kwargs["watch_func"] = self.provider.tail_stack
```

Comment:
This gets removed in this PR, which may not be correct. This seems to tail the events, but I'm not sure I've ever actually seen stacker do this.